crates/solidity/inputs/language/src/lib.rs

```diff
@@ -2,4 +2,40 @@
 
 mod definition;
 
+use codegen_language_definition::model::BuiltIn;
 pub use definition::SolidityDefinition;
+
+pub fn render_built_ins(built_ins: &[BuiltIn]) -> String {
+    let mut lines: Vec<String> = Vec::new();
```

Comment:
suggestion if it is more ergonomic for you: `write!()` and `writeln!()` can be used directly in strings, instead of creating a `Vec<String>` and joining it at the end.
Example in `crates/codegen/spec/src/generators/grammar_ebnf.rs`.

Reply:
I also suggest moving this to a `mod bindings` to maintain the structure.

Reply:
Nice! I didn't know you could `write!` or `writeln!` directly to a string. I'll change this.